README.md

```diff
@@ -1,7 +1,5 @@
 # hello_world
 This is the introductory exercise for GitHub (from https://guides.github.com/activities/hello-world/).
-The guide says to write a bit about myself... I'm a carbon-based life form, or at least, I seem to think I am. Beyond that, I don't seem to have much in common with other carbon-based life forms, except some house cats.
-The areas in which I differ include 1) that I think it's too cold (everywhere, all the time), and 2) that it's been a long time since I've coded regularly.
-That's about to change.
 
-(What did you think cats were doing while you're away? They're coding in higher dimensions.)
```

Comment:
I want this to go below the end of what's in the branch version. I don't see how that can be done...